tasks.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2017 Snowplow Analytics Ltd. All rights reserved.
+#
+# This program is licensed to you under the Apache License Version 2.0,
+# and you may not use this file except in compliance with the Apache License Version 2.0.
+# You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the Apache License Version 2.0 is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+
+import datetime, json, uuid, time
+from functools import partial
+from random import choice
+
+from invoke import run, task
+
+from google.cloud import pubsub
+from google.cloud import bigtable
+import os.path
+
+JAR_FILE = "gcp-dataflow-streaming-example-project-0.1.0.jar"
```

Comment:
`sbt assembly` generates `gcp-dataflow-example-project-0.1.0-rc2.jar` not `gcp-dataflow-streaming-example-project-0.1.0-rc2.jar` (`streaming` is redundant). Also it may be worth to specify full path with `target/scala-2.11`. If user downloaded it from bintray he may specify custom path.